dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -640,8 +640,7 @@ def get_airflow_extras():
 PROVIDERS_COMPATIBILITY_TESTS_MATRIX: list[dict[str, str | list[str]]] = [
     {
         "python-version": "3.9",
-        "airflow-version": "2.9.3",
-        "remove-providers": "cloudant common.messaging fab edge3 git",
+        "airflow-version": "2.10.0",
         "run-tests": "true",
     },
```

Comment:
I think if we do not check compatability with 2.9 we do not need to check against 2.10.0 as well... as we cover this with 2.10.5 in the block below.
So I'd remove the Airflow 2.9 block w/o replacement.

Reply:
I left it as 2.10 is no the same as 2.10.5
Not sure why the 2.10.5 was added?

Reply:
In the past the provider compatability tests were always made against the latest patch of each minor release. So when 2.10.5 was released we changed back-compat tests from 2.10.4 to 2.10.5.